ruby/vampires.rb

```diff
@@ -0,0 +1,55 @@
+Vampire_Names = "dra drac ula cula fang nosf eratu"
+Allergies = %w(peanuts sunshine wheat beestings)
+Employee_Roster = []
+
+
+def vamp_namer(name)
+	vamp_array = Vampire_Names.split(" ")
+	vamp_array.each { |vamp_ele| return true if name.downcase.include?(vamp_ele) }
+	false
+end
+
+puts "How many employees to process?"
+num = gets.chomp.to_i
+
+num.times do
+	employee_allergies = Hash.new(0)
+
+	puts "What is your name?"
+	name = gets.chomp
+
+	puts "How old are you?"
+	age = gets.chomp.to_i
+
+	puts "What year were you born?"
+	birth_year = gets.chomp.to_i
+
+	puts "Our company cafeteria serves garlic bread. Should we order some for you? (y / n)"
+	garlic_order = gets.chomp
+
+	puts "Would you like to enroll in the company’s health insurance? (y / n)"
+	insurance = gets.chomp
+
+	Allergies.each do |allergy|
+		puts "Are you allergic to #{allergy}? (y / n)"
+		response = gets.chomp
+		employee_allergies[allergy] = response
+	end
+
+	case
+	when age < 116 && birth_year > 1900 && !vamp_namer(name)
+		puts "Probably not a vampire... Actually, never mind! What do these questions have to do with anything? Let's all be friends."
+	when (age > 116 || birth_year < 1900) && (garlic_order == "n" || insurance == "n")
```

Comment:
Don't forget to check the employee's age in comparison to what year it actually is. You can compare the age they entered: `current year - age` compared to the year they were born, `birth_year`. Instead of hardcoding the year as 2016 for the `current_year` or checking if it was before a specific year, you can use the Ruby Time module (check it out in the Ruby docs) and use `Time.now` so your program could be used in any year.